### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER and repository) in composer.json.

In addition, the TER homepage is missing the "Packagist" button, which seem to be caused by an empty "Composer name of extension" field in the TER extension edit form.

Relates: TYPO3-Documentation/T3DocTeam#185

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[![Latest Stable Version](https://poser.pugx.org/friendsoftypo3/compatibility6/v/stable.svg)](https://extensions.typo3.org/extension/compatibility6/)
+[![TYPO3 11](https://img.shields.io/badge/TYPO3-7-orange.svg?style=flat-square)](https://get.typo3.org/version/7)
```
Comment (ohader) [inline README.md]: v11??

**Review by ohader — CHANGES_REQUESTED** [review]
Target was TYPO3 v7, not v11

File: README.md (hunk 1 above)
Comment (alexander-nitsche) [reply README.md]: Lost in batch processing. Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
packagist.org reference added in TER

**Pull request merged**